smart-contracts/contracts/RelayBridgeFactory.sol

```diff
@@ -11,7 +12,7 @@ contract RelayBridgeFactory {
   event BridgeDeployed(
     address bridge,
     address indexed asset,
-    address indexed proxyBridge
+    BridgeProxy indexed proxyBridge
```

Comment:
Why pass a contract object instead of an address here ?

Reply:
cleaner?